Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:8.2-cli-alpine
+
+ARG UID=1000
+ARG GID=1000
+
+# Configure non-root user and group
+RUN addgroup -g $GID appgroup
+RUN adduser -u $UID -G appgroup -s /bin/sh -D appuser
+
+# Instala Composer
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
+
+# Instala las dependencias necesarias
+RUN apk add --no-cache libpng-dev libjpeg-turbo-dev freetype-dev libzip-dev icu-dev
+
+ENV PATH="/usr/app/vendor/bin:/usr/app/bin:${PATH}"
+
+# Habilita las extensiones
+RUN docker-php-ext-install gd zip intl opcache
```

Comment:
This one I would like to change. Can you please adopt the same we are doing on Archetype-Symfony?